rfcs/text/0008-email.md

```diff
@@ -0,0 +1,144 @@
+# 0008: Email
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **1** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **Oct 5th 2020** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+This RFC proposes a new top-level field to facilitate email use cases. 
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+## Fields
+
+<!--
+Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
+-->
+
+| field | type | description |
+| --- | --- | --- |
+| `email.action` | keyword | Action take by the source device, e.g. delivered, blocked, quarantined, deleted |
+| `email.bcc.address` | keyword | Addresses of Bcc's |
+| `email.bcc.domain` | keyword | Domains of the Bcc's |
+| `email.cc.address` | keyword | Addresses of Cc's |
+| `email.cc.domain` | keyword | Domains of Cc addresses |
```

Comment:
@webmat I think it would make more sense to add `related.email` and maybe also `related.domain` instead. It would conform more to the current structure of how we handle arrays like this without nesting too much.
What do you think?

Reply:
Under the current guidance with the `related.*` fields, email addresses are captured in `related.user` and domains in `related.hosts`.

Rather we continue with this guidance or adjust and add additional `related.*` fields is a good conversation to have. 🤔